### ISSUE TITLE
模型权重文件

### ISSUE THREAD

**Issue by orange998** [PANPEIWEN/ABC#5]
你好，读了你的论文后，我受到了一些启发，但是在做实验复现时结果有点不太理想，可以提供下模型网络关于每个数据集的权重文件嘛，best.pth.tar

**Comment by PANPEIWEN** [comment]
你好 你能把你的训练细节说一下嘛 我这两天就会把训练好的模型权重整理好

orange998 ***@***.***> 于2023年4月13日周四 20:26写道：

**Comment by orange998** [comment]
data = dict(base_size=256,
    crop_size=256,
    data_aug=True,
    suffix='png',
    num_workers=8,
    train_batch=32,
    test_batch=16,
    train_dir='trainval',
    test_dir='test')
log_config = dict(
    interval=10,
    hooks=[
        dict(type='TextLoggerHook', by_epoch=True),
        dict(type='TensorboardLoggerHook')
    ])
dist_params = dict(backend='nccl')
log_level = 'INFO'
load_from = None
resume_from = None
workflow = [('train', 1)]
cudnn_benchmark = True
gpus = 1
optimizer = dict(
    type='AdamW',
    setting=dict(lr=0.0001, weight_decay=0.01, betas=(0.9, 0.999)))
optimizer_config = dict()
lr_config = dict(
    policy='PolyLR',
    warmup='linear',
    power=0.9,
    min_lr=0.0001,
    warmup_epochs=5)
runner = dict(type='EpochBasedRunner', max_epochs=1500)
checkpoint_config = dict(by_epoch=False, interval=1)
evaluation = dict(epochval=1, metric='mIoU', pre_eval=True)
model = dict(
    name='ABCNet',
    type='EncoderDecoder',
    pretrained=None,
    backbone=dict(type=None),
    decode_head=dict(
        type='ABCNet',
        in_ch=3,
        out_ch=1,
        dim=16,
        ori_h=256,
        deep_supervision=True),
    loss=dict(type='SoftIoULoss'))
我没有对默认文件做改动，只更改了epochs，这是训练nudt数据集的信息

**Comment by PANPEIWEN** [comment]
你好 你使用的是abcnet_clft-s_256x256_500e_nudt.py
<https://github.com/PANPEIWEN/ABC/blob/main/configs/abcnet/abcnet_clft-l_256x256_500e_nudt.py>这个文件
这个文件对应的模型最小 性能也是最差的 我们的最好结果是用abcnet_clft-l_256x256_500e_nudt.py
<https://github.com/PANPEIWEN/ABC/blob/main/configs/abcnet/abcnet_clft-l_256x256_500e_nudt.py>
这个文件训练出来的
另外 这个数据集收敛的较慢 并且我们的学习率衰减策略是根据epochs来决定的 所以建议你保持原来的epochs

orange998 ***@***.***> 于2023年4月15日周六 15:24写道：

**Comment by orange998** [comment]
好的，十分感谢

**Comment by RenYong1** [comment]
楼主，可以提供一下你的环境配置相关包的信息吗？我是windows系统，代码调了很久都调不通

**Comment by orange998** [comment]
你可以在云服务器上试一下环境，我记得我的cuda是11.3，pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 cudatoolkit=11.3，其他的缺什么直接装的

**Comment by RenYong1** [comment]
好的，十分感谢，再请教一下你是linux系统还是windows系统捏？

**Comment by PANPEIWEN** [comment]
> 好的，十分感谢，再请教一下你是linux系统还是windows系统捏？

是Linux教程

**Comment by RenYong1** [comment]
好的，感谢指教
